src/command_modules/azure-cli-extension/setup.py

```diff
@@ -14,7 +14,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "0.0.9"
+VERSION = "0.0.10"
```

Comment:
No need to change version. Someone already updated it.
The current on PyPI is 0.0.8 so 0.0.9 is fine.
https://pypi.python.org/pypi/azure-cli-extension